src/jurisdictions.ts

```diff
@@ -0,0 +1,108 @@
+// https://worldwide.espacenet.com/patent/help/countrycodes
```

Comment:
Potentially insufficient country code list. Might need to replace with:
https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2#Officially_assigned_code_elements

Will require addition of custom name for jurisdictions which are not countries (WIPO, EPO, etc).